goon/browserassets/js/browserOutput.js

```diff
@@ -72,6 +72,7 @@ var opts = {
 // If not set to normal, a CSS file `browserOutput_${name}.css` will be added to the head.
 var colorPresets = [
 	'normal',
+	'dark',
```

Comment:
You should add this to the end of this list. The color preset used is stored as integer into this list so adding it in the middle will mean people who are currently set to the color blind preset will get automatically switched to dark.

Reply:
I realize this but intentionally broke it anyways under the reasoning of:

1- Only 2 people ever must be on colorblind mode, and if they are in colorblind mode then they already know how to change color presets so as to get back to it again (or they may even find darkmode is colorblind friendly as well and not need to switch at all)
2- I want darkmode to be #2 since it makes more sense and is easier to find and this alone is worth mildly inconveniencing those 2 people

Reply:
Fair enough.